test/v1/retrieveForeignReliefs/def1/model/response/Def1_ForeignIncomeTaxCreditReliefSpec.scala

```diff
@@ -14,14 +14,13 @@
  * limitations under the License.
  */
 
-package v1.models.response.retrieveForeignReliefs
+package v1.retrieveForeignReliefs.def1.model.response
```

Comment:
Nice one on adding the tests for this endpoint as well, can we add one for the request side as well if it exists? If not this is still fine